client/signup/steps/user/index.jsx

```diff
@@ -146,6 +146,10 @@ export class UserStep extends Component {
 			subHeaderText = '';
 		}
 
+		if ( positionInFlow === 0 && [ 'onboarding', 'onboarding-dev' ].includes( flowName ) ) {
```

Comment:
In this case, the position === 0 does seem to make sense but not sure about the flow name conditions.

Reply:
Similarly to the above, some flows won't ever really have conditions that are suitable for the site creation copy we're using.

We can definitely add flows to this list or invert it to a block list going forward.

Reply:
ok cool, makes sense. Thanks for clarifying.